src/memo/memo.ml

```diff
@@ -916,12 +922,17 @@ module Stack_frame = struct
     Option.map t.spec.human_readable_description ~f:(fun f -> f t.input)
 end
 
+let invalidate_store =
+  Store.iter ~f:(fun node -> node.Dep_node.last_cached_value <- None)
```

Comment:
May be nice to share this line of code with `invalidate_dep_node`, so we have a reminder to update both places in sync when we need to update them.

Reply:
Good idea.

Reply:
Done.